charts/tiller.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
Last question, probably due lack of experience with helm - why this file is needed? For me everything was working without it.

Reply:
I just conventionally have been doing this because I had tiller service account issue https://github.com/fnproject/fn-helm/issues/21.
I've just tested without this, it seems working fine without creating special service account!
deleted `tiller.yaml`